tracing-journald/src/lib.rs

```diff
@@ -257,12 +273,19 @@ fn put_metadata(buf: &mut Vec<u8>, meta: &Metadata, span: Option<usize>) {
     }
 }
 
-fn put_debug(buf: &mut Vec<u8>, name: &str, value: &dyn fmt::Debug) {
+fn put_field_value(buf: &mut Vec<u8>, name: &str, value: FieldValue<'_>) {
```

Comment:
To save the enum and monomorphize a little more, how about:
```suggestion
fn put_field_value(buf: &mut Vec<u8>, name: &str, formatter: impl FnOnce(&mut Vec<u8>)) {
```

Reply:
Done.